From b0afae7500ecf2a98b5055ecfac4d372796e69ce Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Thu, 18 Dec 2014 11:59:01 -0500
Subject: [redhat] kabi: Adding symbol __request_region (kernel/resource.c)

Message-id: <dfe258df5114519e54408ffadcf7c59da98e48e5.1418903403.git.poros@redhat.com>
Patchwork-id: 102239
O-Subject: [PATCH RHEL7.1 09/48] kabi: Adding symbol __request_region (kernel/resource.c)
Bugzilla: 1160760
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1160760
    https://bugzilla.redhat.com/show_bug.cgi?id=1160760

    Description:
    ============
    These are called by request_mem_region() and
    release_mem_region() macros which are used by
    basically every PCI-driver.
    Used in ioc4_extint.ko

Signed-off-by: Petr Oros <poros@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/__request_region b/redhat/kabi/kabi-module/kabi_ppc64/__request_region
new file mode 100644
index 0000000..0ff8be6
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/__request_region
@@ -0,0 +1,2 @@
+#1-
+0x1fedf0f4 __request_region vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_ppc64le/__request_region b/redhat/kabi/kabi-module/kabi_ppc64le/__request_region
new file mode 100644
index 0000000..0ff8be6
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64le/__request_region
@@ -0,0 +1,2 @@
+#1-
+0x1fedf0f4 __request_region vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/__request_region b/redhat/kabi/kabi-module/kabi_s390x/__request_region
new file mode 100644
index 0000000..0ff8be6
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/__request_region
@@ -0,0 +1,2 @@
+#1-
+0x1fedf0f4 __request_region vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/__request_region b/redhat/kabi/kabi-module/kabi_x86_64/__request_region
new file mode 100644
index 0000000..0ff8be6
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/__request_region
@@ -0,0 +1,2 @@
+#1-
+0x1fedf0f4 __request_region vmlinux EXPORT_SYMBOL
-- 
1.7.1